### PR TITLE
Chapter 6 challenge output class

### LINKED ISSUE #768: Chapter 6 Challenge 1 Output Class
Implement the second challenge page for chapter 6

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=6200%3A31354&mode=dev

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #768 

Although  haven't gotten through the validation yet

### CODE REVIEW

File: content/lessons/chapter-6/in-out-4.tsx
Comment (benalleng) [inline content/lessons/chapter-6/in-out-4.tsx:63]: Lets change the validation to use Matthew's challenge code for consistency

https://github.com/saving-satoshi/challenges/blob/master/chapter6/javascript/challenge.js#L66

otherwise looking good! almost there!

**Review by benalleng — CHANGES_REQUESTED** [review]

File: content/lessons/chapter-6/in-out-4.tsx
Comment (benalleng) [inline content/lessons/chapter-6/in-out-4.tsx:131]: We can use the new import now

**Review by daviroo — CHANGES_REQUESTED** [review]

**Comment by daviroo** [comment]
@0tuedon we added the bech32 libs as backend dependencies in this PR https://github.com/saving-satoshi/saving-satoshi-backend/pull/33/.  Also it looks like you will need to rebase when you get a chance as there is a merge conflict.

File: ui/lesson/ScriptingChallenge/library/bech32js.ts
Comment (0tuedon) [reply ui/lesson/ScriptingChallenge/library/bech32js.ts:1]: yh i guess i used that but the file remains, would rebase

**Comment by 0tuedon** [comment]
> @0tuedon we added the bech32 libs as backend dependencies in this PR [saving-satoshi/saving-satoshi-backend#33](https://github.com/saving-satoshi/saving-satoshi-backend/pull/33). Also it looks like you will need to rebase when you get a chance as there is a merge conflict.

done.

**Review by daviroo — APPROVED** [review]

**Comment by daviroo** [comment]
@0tuedon I think you will either need to get @benalleng to approve or dismiss his stale review before we can merge this.

**Review by benalleng — APPROVED** [review]
Tested! LGTM

**Pull request merged**
